index.html

```diff
@@ -1,22 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <!-- Title -->
-    <link rel = "icon" 
-        href = "./assets/images/leaf.png" 
-        type = "image/x-icon">
+    <link rel="icon" href="./assets/images/leaf.png" type="image/x-icon" />
```

Comment:
Instead of static paths can we use relative paths?
**Note: When you are given a review and you accepted to change it. Please try to change it from everywhere, cause the reviewer may not be able to review each line with the same required changes.**